File: app/src/main/java/com/bopr/android/smailer/ui/LegalInfoActivity.java
package com.bopr.android.smailer.ui;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;

/**
 * An activity that shows legal info.
 *
 * @author Boris Pronin (<a href="mailto:boprsoft.dev@gmail.com">boprsoft.dev@gmail.com</a>)
 */
public class LegalInfoActivity extends AppActivity {

    @NonNull
    @Override
    protected Fragment createFragment() {
        return new LegalInfoFragment();
    }

}

File: app/src/main/kotlin/com/bopr/android/smailer/ui/LegalInfoActivity.kt
package com.bopr.android.smailer.ui

import androidx.fragment.app.Fragment

/**
 * An activity that shows legal info.
 *
 * @author Boris Pronin ([boprsoft.dev@gmail.com](mailto:boprsoft.dev@gmail.com))
 */
class LegalInfoActivity : BaseAppActivity() {

    override fun createFragment(): Fragment {
        return LegalInfoFragment()
    }
}